en/hosting_guide/site_creation.rst

```diff
@@ -0,0 +1,106 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires three steps. Additionnally, you can add some more
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with rights
+* Contributing the home page
+* Creating an Apache virtual host
```

Comment:
Officially Open Orchestra requires an Apache server, but you're right, I'll try to be as transparent as possible